gno.land/cmd/gnoweb/main.go

```diff
@@ -130,12 +133,26 @@ func (c *webCfg) RegisterFlags(fs *flag.FlagSet) {
 		"nable privacy-first analytics",
```

Comment:
Not in your changes but just noticed this typo 
```suggestion
		"enable privacy-first analytics",
```